massa-node/base_config/config.toml

```diff
@@ -1,6 +1,6 @@
 [logging]
     # Logging level. High log levels might impact performance. 0: ERROR, 1: WARN, 2: INFO, 3: DEBUG, 4: TRACE
-    level = 2
+    level = 3
```

Comment:
Left from your testing sessions ?

Reply:
Yes, reverted this in 491b0d8b41692461791c07801372fdde5d3083df